test/NuGet.Core.Tests/NuGet.Packaging.Test/PackagesConfigWriterTests.cs

```diff
@@ -344,5 +344,87 @@ public void PackagesConfigWriter_OpenExistingFile()
             var packageNode = xml.Descendants(PackagesConfig.PackageNodeName).FirstOrDefault();
             Assert.Equal(packageNode.ToString(), "<package id=\"packageA\" version=\"1.0.1\" targetFramework=\"net45\" userInstalled=\"true\" protocolVersion=\"V2\" />");
         }
+
+        [Fact]
+        public void PackagesConfigWriter_ThrowOnMalformedPackagesConfigXml()
+        {
+            // Arrange
+            var folderPath = Path.GetTempPath() + Convert.ToString(DateTime.Now.Ticks % 2000);
```

Comment:
Get rid of the convert.ToString().

Simply create a Guid, instead of a temp path that doesn't get cleaned up later.